OpenSim/Common/Reporter.h

```diff
@@ -163,10 +163,24 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
     TableReporter_() = default;
     virtual ~TableReporter_() = default;
 
+    /** Retrieve the report.                                                  */
```

Comment:
The class documentation does say what the template argument `ValueT` is in one sentence.
Providing examples in documentation could be topic of another PR. I have not done anything about this here.